src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/UsersController.cs

```diff
@@ -93,12 +93,44 @@ public async Task<ActionResult<GetUser>> GetUserById(string userId)
             {
                 Name = user.DisplayName,
                 PhoneNumber = user.PhoneNumber,
-                EmailAddress =  user.Email,
+                EmailAddress = user.Email,
                 Disabled = user.Disabled,
                 PrimaryOrganisationId = user.PrimaryOrganisationId
             };
 
             return Ok(getUser);
         }
+
+        [Route("api/v1/users/{userid}/enable")]
+        [HttpPost]
+        public async Task<ActionResult> EnableUserAsync(string userId)
+        {
+            var user = await _usersRepository.GetUserByIdAsync(userId);
+
+            if (user is null)
+            {
+                return NotFound();
+            }
+
+            user.MarkAsEnabled();
+            await _usersRepository.UpdateAsync(user);
+            return NoContent();
+        }
+
+        [Route("api/v1/users/{userid}/disable")]
+        [HttpPost]
```

Comment:
Same as comment above ^

Reply:
I'm also curious about the two separate endpoints – is that what the FE wanted? (I would've thought a single endpoint would be easier for them.)

Reply:
This was the discussion @bjss-chris https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_sprints/taskboard/Buying%20Catalogue%20Tech%20Cell/Buying%20Catalogue/Iteration%2021?workitem=5623

Reply:
Perhaps it's just me, but that discussion thread doesn't make much sense. However, I'll trust that this does whatever was decided (for whatever reason) as linked above.

We do currently have quite a bit of duplication between the two methods though – I think there is only one line that differs between them. Perhaps they could be refactored to reduce the duplication?